src/tests/work-item/work-item-list/page-objects/work-item-list.page.js

```diff
@@ -276,13 +276,37 @@ class WorkItemListPage {
   get filterDropdown () {
     return  element(by.id("wi_filter_dropdown"));
   }
-  filterDropdownId () {
```

Comment:
I discovered this duplicated function during this PR , it looks like never used so removed it :)